src/functions.jsx

```diff
@@ -0,0 +1,3 @@
+export function findSelectedTab(tabse, selectedTabId) {
```

Comment:
The function argument 'tabse' name is not clear, it's better to rename it to 'tabs' for better readability.